tests/modules/gatk4/mergebamalignment/main.nf

```diff
@@ -6,11 +6,11 @@ include { GATK4_MERGEBAMALIGNMENT } from '../../../../modules/gatk4/mergebamalig
 
 workflow test_gatk4_mergebamalignment {
     input    = [ [ id:'test' ], // meta map
-                 file(params.test_data['sarscov2']['illumina']['test_single_end_bam'], checkIfExists: true)
+                 file(params.test_data['sarscov2']['illumina']['test_single_end_bam'], checkIfExists: true),
```

Comment:
As per usual my question is: does this work with cram files too?

Reply:
From what I have seen in both the GATK and picard tool documentations, I don't think it works with cram files as neither of them mention cram. Is there an example of an unaligned bam (from what I understand you don't get unaligned crams)  and an aligned cram we could use to double check on the test datasets repo?

Reply:
I have seen some discussions around https://www.biostars.org/p/389788/ https://davetang.org/muse/2022/01/20/storing-fastq-as-unaligned-cram/ but not seen it out in the wild. For me it is fine to leave it as is